src/Map.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import mapboxgl from 'mapbox-gl';
+import './Map.css';
+import { statesData } from './us-states.js';
+import { MAPBOX_TOKEN } from './tokens.js';
+
+mapboxgl.accessToken = MAPBOX_TOKEN;
+
+class Map extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
Is this setting a default state?

Reply:
Oh it's in the package.json file. Since you add semantic as dependency and I add mapbox-gl as dependency. They occupy the same space in the file and git don't know which to include. I resolved the conflict locally to include both

Reply:
And for the state, it's a var owned by Map component. It basically tells you what is the default center and zoom level of the map

Reply:
Okay cool, thanks!